websites/P/Podurama/metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "358402930191106049",
+		"name": "ion606"
+	},
+	"service": "Podurama",
+	"description": {
+		"en": "Google Podcast presence, but with the podcast artwork"
+	},
+	"url": "podurama.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/CNkpXlB.png",
+	"thumbnail": "https://i.imgur.com/CNkpXlB.png",
```

Comment:
The [thumbnail](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#thumbnail) should preferably be a [wide promotional card](https://i.imgur.com/3QfIc5v.jpg) or a [screenshot](https://i.imgur.com/OAcBmwW.png) if the first is not available.